### ISSUE TITLE
Issues for R180

### ISSUE THREAD

**Issue by craigsapp** [polyrhythm-project/rds-scores#77]

**Comment by rafaperlau** [comment]
- pink notation shown when trying to open data in humdrum

**Comment by epoudrier** [comment]
Specify which method was used to import the kern notation. Did you try to quick way?

**Comment by rafaperlau** [comment]
Copying raw data shows pink notation. Opening from Verovio worked though.

**Comment by rafaperlau** [comment]
1. Added hanging tie at trumpet in C m. 130, but it is not showing (Humdrum).
2. Hanging slur at trumpet in C m. 130 is starting at the first note instead (Humdrum).
3. Tuplet on Violin I m. 131 is showing (Sibelius).
4. Tuplets (number 4) on Violins and violas are showing (Sibelius).
5. Tuples on violins and violas m. 130 are not showing (Humdrum).
6. Missing rehearsal number (Humdrum)

**Comment by epoudrier** [comment]
@rafaperlau Are items 2 to 6 still outstanding?

**Comment by rafaperlau** [comment]
Yes. Should I fix the issues in Sibelius?

2. From what I read from the Humdrum powerpoint, those slur issues are not able to be fixed at this point.
6. The same with rehearsal numbers.
5. I also tried to figure out a way to show those tuplets, but I was not able to. I fixed the ones at the clarinet, but not those, since they are hidden in Humdrum.

**Comment by epoudrier** [comment]
No, we will do this later. Thanks! In future posting, it might be good to divide these in two (edited in Humdrum vs. outstanding) so we can tell more easily what needs to be done. Sibelius edits will be addressed later.

**Comment by rafaperlau** [comment]
Update:

Fixed:
1. Changed tuplets on Clarinet in A bars 130, 131
2. Added hanging tie at trumpet in C m. 130, but it is now showing
3. Hanging slur at trumpet in C m. 130
4. Add "simile" to violins II and viola m. 131
5. Add rehearsal number at bar 130
6. Add intrument codes

Pending:
7. Tuples on violins and violas m. 130 are not showing (Humdrum).
8. Tuplet on Violin I m. 131 is showing (Sibelius).
9. Tuples (4) on Violins and violas are showing (Sibelius).
10. Rehearsal number at bar 130 is not showing (Humdrum)

**Comment by epoudrier** [comment]
No. 5 and no. 10 are contradictory. Clarify.

**Comment by epoudrier** [comment]
Ares nos. 8 and 9 missing from Humdrum?

**Comment by rafaperlau** [comment]
> No. 5 and no. 10 are contradictory. Clarify.

I added the rehearsal number, but it is not showing in Humdrum. I showld have put it only in "pending" section.

**Comment by rafaperlau** [comment]
> Ares nos. 8 and 9 missing from Humdrum?

No. 8 and 9 are correctly hidden in Humdrum, but it is showing in Sibelius. Sibelius file needs to be edited.

**Comment by aaronearlgraham** [comment]
ROUND 2***
1. Tempo marking needs brackets added
2. Rehearsal mark needs to be added
3. CLRNT in A - mm 2 - notes and rest clash with each other (visually)
4. BSN - mm 3 - slur needs to be changed to hanging slur
5. HRNS - mm 2 - added 2nd and 4th text markings
6. TRPT mm 1 - slur needs to be changed to hanging slur
7. Brass bracket needs to be adjusted to include TRPT
8. Sub brackets need to be added to string parts

**Comment by cdf75ubc** [comment]
Dr. @epoudrier : 
1) This file has quintuples wrongfully notated from SIB. They are 5 8th notes in the place of 3 (3/8). It was notated as 5 **dotted** 8th notes in the place of 3. screenshots below
2) This file has wrongfully notated triplet tremolos in the strings from SIB. Sounds the same but do not look the same, besides missing tuplet number and articulation. screenshots below

original
![image](https://user-images.githubusercontent.com/59900770/95761883-55859900-0c83-11eb-8cdd-192e2c5270ef.png)


SIB/PDF/VHV
![image](https://user-images.githubusercontent.com/59900770/95761981-7cdc6600-0c83-11eb-92ae-c1f4e406bb6a.png)

**Comment by cdf75ubc** [comment]
also rehearsal number is wrong: the mark (17) is on top of bar 126 (not in the excerpt).

**Comment by cdf75ubc** [comment]
1. tempo  marking brackets added
2. rehearsal mark deleted
3. slurcheck has been aplied properly
4. system decoration has been improved]
5. wrong quintuplet notation has been corrected
6. articulations have been added to upper strings
